15-7-1_and_15-c-1/update_15-7-1_and_15-c-1.R

```diff
@@ -0,0 +1,90 @@
+  rename_column(primary = "Number of Times seized: Live Coral & Coral Derivatives", new_name = "Live Coral and Coral Derivatives") %>% 
+  rename_column(primary = "Number of Times seized: Ivory and Items Containing Ivory", new_name = "Ivory and Items Containing Ivory") %>% 
+  rename_column(primary = "Number of Times seized: Live Animals and Birds", new_name = "Live Animals and Birds") %>%
+  rename_column(primary = "Number of Times seized: Live Plants", new_name = "Live Plants") %>%
+  rename_column(primary = "Number of Times seized: Parts or Derivatives of Animals/Birds", new_name = "Parts or Derivatives of Animals or Birds") %>%
+  rename_column(primary = "Number of Times seized: Parts or Derivatives of Plants", new_name = "Parts or Derivatives of Plants") %>%
+  rename_column(primary = "Number of Times seized: Timber or Wood Products", new_name = "Timber or Wood Products") %>%
+  rename_column(primary = "Number of Times seized: Preparations Of Oriental Medicine Which Include Parts & Derivatives Of Endangered Species", new_name = "Preparations Of Oriental Medicine Which Include Parts and Derivatives Of Endangered Species") %>%
+  rename_column(primary = "Number of Times seized: Butterflies", new_name = "Butterflies") 
+
+# split year and quarter column into two
+
+renamed_seizures_main_data[c('Year', 'Quarter')] <- str_split_fixed(renamed_seizures_main_data$Quarter, ' ', 2)
+
+# select wanted columns
+
+renamed_seizures_main_data <- renamed_seizures_main_data %>% 
+  select(c("Year", "Quarter", "Total Number of Seizures", "Caviar and Caviar extract", "Live Coral and Coral Derivatives", "Ivory and Items Containing Ivory", "Live Animals and Birds", "Live Plants", "Parts or Derivatives of Animals or Birds", "Parts or Derivatives of Plants", "Timber or Wood Products", "Preparations Of Oriental Medicine Which Include Parts and Derivatives Of Endangered Species", "Butterflies"))
+
+# remove rows containing only NAs
+
+renamed_seizures_main_data <- na.omit(renamed_seizures_main_data)
+
+# change ":" to NAs
+
+renamed_seizures_main_data[renamed_seizures_main_data == ":"] <- "NA" 
+
+# calculate total seizures for each import type for each year
+
+seizures_main_data_totals <-  renamed_seizures_main_data %>% 
+  select(c("Year", "Total Number of Seizures", "Caviar and Caviar extract", "Live Coral and Coral Derivatives", "Ivory and Items Containing Ivory", "Live Animals and Birds", "Live Plants", "Parts or Derivatives of Animals or Birds", "Parts or Derivatives of Plants", "Timber or Wood Products", "Preparations Of Oriental Medicine Which Include Parts and Derivatives Of Endangered Species", "Butterflies"))
+
+# change column names to sentence case
+
+seizures_main_data_totals <- seizures_main_data_totals %>%
+  rename_with(str_to_sentence)
+
+# format into csv and remove NAs
+
+seizures_main_data_totals <- seizures_main_data_totals %>%
+  pivot_longer(-c("Year"), names_to = "Import type", values_to = "Value")
+
```

Comment:
You can add a line here to retain a variable with the NAs, to be used later for the QA report, for example:
`select_NAs <- seizures_main_data_totals %>%
  filter(Value == "NA")`

Reply:
Done